functions-and-operators/tidb-functions.md

```diff
@@ -573,4 +579,118 @@ SELECT VITESS_HASH(123);
 | 1155070131015363447 |
 +---------------------+
 1 row in set (0.00 sec)
-```
\ No newline at end of file
+```
+
+## TIDB_ENCODE_INDEX_KEY
+
+Encodes an index key.
+
+```sql
+CREATE TABLE t(id int PRIMARY KEY, a int, KEY `idx` (a));
+```
+
+```
+Query OK, 0 rows affected (0.00 sec)
+```
+
+```sql
+INSERT INTO t VALUES(1,1);
+```
+
+```
+Query OK, 1 row affected (0.00 sec)
+```
+
+```sql
+SELECT TIDB_ENCODE_INDEX_KEY('test', 't', 'idx', 1, 1);
+```
+
+```
++----------------------------------------------------------------------------+
+| TIDB_ENCODE_INDEX_KEY('test', 't', 'idx', 1, 1)                            |
++----------------------------------------------------------------------------+
+| 74800000000000007f5f698000000000000001038000000000000001038000000000000001 |
++----------------------------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+## TIDB_ENCODE_RECORD_KEY
+
+Encodes a record key.
+
+```sql
+CREATE TABLE t(id int PRIMARY KEY, a int, KEY `idx` (a));
+```
+
+```
+Query OK, 0 rows affected (0.00 sec)
+```
+
+```sql
+INSERT INTO t VALUES(1,1);
+```
+
+```
+Query OK, 1 row affected (0.00 sec)
+```
+
+```sql
+SELECT TIDB_ENCODE_RECORD_KEY('test', 't', 1);
+```
+
+```
++----------------------------------------+
+| TIDB_ENCODE_RECORD_KEY('test', 't', 1) |
++----------------------------------------+
+| 7480000000000000845f728000000000000001 |
++----------------------------------------+
+1 row in set (0.00 sec)
+```
+
+```sql
+SELECT TIDB_DECODE_KEY('7480000000000000845f728000000000000001');
+```
+
+```
++-----------------------------------------------------------+
+| TIDB_DECODE_KEY('7480000000000000845f728000000000000001') |
++-----------------------------------------------------------+
+| {"id":1,"table_id":"132"}                                 |
++-----------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+## TIDB_MVCC_INFO
+
+Returns MVCC info for a key. The `TIDB_ENCODE_INDEX_KEY` function can be used to obtain a key.
```

Comment:
```suggestion
Returns the MVCC information for a key. You can use the [`TIDB_ENCODE_INDEX_KEY`](#TIDB_ENCODE_INDEX_KEY) function to obtain a key.
```

Reply:
Same for this one, we could link to https://docs.pingcap.com/tidb/stable/glossary#multi-version-concurrency-control-mvcc